quinn-proto/src/config.rs

```diff
@@ -25,27 +25,27 @@ use crate::{
 /// performance at lower bandwidths and latencies. The default configuration is tuned for a 100Mbps
 /// link with a 100ms round trip time.
 pub struct TransportConfig {
-    pub(crate) max_concurrent_bidi_streams: VarInt,
-    pub(crate) max_concurrent_uni_streams: VarInt,
-    pub(crate) max_idle_timeout: Option<VarInt>,
-    pub(crate) stream_receive_window: VarInt,
-    pub(crate) receive_window: VarInt,
-    pub(crate) send_window: u64,
-
-    pub(crate) max_tlps: u32,
-    pub(crate) packet_threshold: u32,
-    pub(crate) time_threshold: f32,
-    pub(crate) initial_rtt: Duration,
-    pub(crate) initial_max_udp_payload_size: u16,
-
-    pub(crate) persistent_congestion_threshold: u32,
-    pub(crate) keep_alive_interval: Option<Duration>,
-    pub(crate) crypto_buffer_size: usize,
-    pub(crate) allow_spin: bool,
-    pub(crate) datagram_receive_buffer_size: Option<usize>,
-    pub(crate) datagram_send_buffer_size: usize,
-
-    pub(crate) congestion_controller_factory: Box<dyn congestion::ControllerFactory + Send + Sync>,
+    pub max_concurrent_bidi_streams: VarInt,
```

Comment:
Why do we need this? This seems like a blocker for me -- if we need to make all the currently crate-private API public for this, I don't think this idea is workable.

Reply:
The tests call these methods directly and now that the tests are in a separate crate, I assume that means they have to be public (although I'm still pretty new to rust). I'm definitely open to alternatives, if there is a better approach.